src/components/hocs/db/withSpaceById.ts

```diff
@@ -15,7 +15,7 @@ export const withSpaceById = <T extends Attributes>(
 ) => {
   const WithSpaceById = (props: Props<T>) => {
     const { space, spaceId, worldId, isLoaded, isLoading } = useSpaceById(
-      props.spaceId
+      props
```

Comment:
I think these `withSpaceById` and `WithSpaceById` are not used anywhere, and if that's the case, can you please delete it before merge? There are other ways of making HoCs that are easier to manage.